type_specific_codegen.go

```diff
@@ -135,6 +135,9 @@ func (v *Value) MSI(optionalDefault ...map[string]interface{}) map[string]interf
 	if s, ok := v.data.(map[string]interface{}); ok {
 		return s
 	}
+	if s, ok := v.data.(Map); ok {
```

Comment:
Cloud you add a todo comment to find a better solution?

Reply:
Done